dotnetv3/S3/scenarios/TransferUtilityBasics/TransferUtilityBasics/TransferBasics.cs

```diff
@@ -77,49 +98,62 @@
 if (success)
 {
     Console.WriteLine($"Downloaded the files in {bucketName} to {downloadPath}.");
-    Console.WriteLine($"{downloadPath} now contains the fillowing files:");
+    Console.WriteLine($"{downloadPath} now contains the following files:");
     DisplayLocalFiles(downloadPath);
 }
 
 Console.WriteLine("\nThe TransferUtility Basics application has completed.");
 PressEnter();
 
-static void DisplayInstructions()
+// Displays the title for a section of the scenario.
+static void DisplayTitle(string titleText)
 {
-    var sepBar = new string('-', 80);
+    var sepBar = new string('-', Console.WindowWidth);
 
-    Console.Clear();
     Console.WriteLine(sepBar);
-    Console.WriteLine(CenterText("Amazon S3 Transfer Utility Basics"));
+    Console.WriteLine(CenterText(titleText));
     Console.WriteLine(sepBar);
+}
+
+// Displays a description of the actions to be performed by the scenario.
+static void DisplayInstructions()
+{
+    var sepBar = new string('-', Console.WindowWidth);
+
+    DisplayTitle("Amazon S3 Transfer Utility Basics");
     Console.WriteLine("This program shows how to use the Amazon S3 Transfer Utility.");
     Console.WriteLine("It performs the following actions:");
-    Console.WriteLine("\t1. Upload a single object to an S3 bucket.");
-    Console.WriteLine("\t2. Upload all an entire directory from the local computer to an\n\t  S3 bucket.");
-    Console.WriteLine("\t3. Download a single object from an S3 bucket.");
-    Console.WriteLine("\t4. Download the objects in an S3 bucket to a local directory.");
+    Console.WriteLine("\t1. Upload a single object to an Amazon S3 bucket.");
+    Console.WriteLine("\t2. Upload an entire directory from the local computer to an\n\t  Amazon S3 bucket.");
+    Console.WriteLine("\t3. Download a single object from an Amazon S3 bucket.");
+    Console.WriteLine("\t4. Download the objects in an Amazon S3 bucket to a local directory.");
     Console.WriteLine($"\n{sepBar}");
 }
 
+// Pauses the scenario.
 static void PressEnter()
 {
     Console.WriteLine("Press <Enter> to continue.");
     _ = Console.ReadLine();
     Console.WriteLine("\n");
 }
 
+// Returns the string textToCenter, padded on the left with spaces
+// that will cause the text to be centered on the console display.
```

Comment:
that will cause the text to be centered --> that center the text

Reply:
changed